### ISSUE TITLE
MultiClusterApp UI :- Nothing happens on multi cluster app answers change 

### ISSUE THREAD

**Issue by kinarashah** [rancher/rancher#18130]
Rancher master: 2/14 

Create multi cluster app. 
On upgrade, change any answer

Expected:
Upgrade goes through, similar to edit. 

Actual:
Page stays the same

**Comment by sangeethah** [comment]
The same issue is also seen when any answer overrides are done.

**Comment by davidnuzik** [comment]
@sangeethah #18196 is preventing me from seeing answers yaml preview in UI FYI so I skipped this for now. I just validated by doing the following which should be sufficient.

Version: Master 2.2 (2/19/19)
- Create MC app, inspect HTTP POST (create) and API answers values
- Upgrade (Edit) MC app, inspect HTTP PUT (edit) and API answers values.
-- Ensure my changed answer is there (it's also a secret). Ensure the secret changed.
- Upgrade again but with answer overrides. inspect HTTP PUT (edit) and API answers values.
-- Confirmed the overrides worked as well.
